app/models/rent.rb

```diff
@@ -7,6 +7,6 @@ class Rent < ApplicationRecord
   belongs_to :book
 
   def valid_date
-    return errors.add(:to, "can't be before from") if from > to
+    errors.add(:to, "can't be before from") if from > to
```

Comment:
Errors should be as descriptive as possible

Reply:
Do you have any standard for messages in these validations?

Reply:
Ok